content/en/hub/initial-setup/storage/encryption.md

```diff
@@ -113,24 +113,45 @@ The dataset listing changes to show the unlocked icon.
 
 ## Conversion from GELI
 
-It is not possible to convert an existing FreeNAS/TrueNAS 11.3 or earlier GELI-encrypted pool to use native ZFS encryption.
-However, data can be migrated from the GELI-encrypted pool to a new ZFS-encrypted pool. The new ZFS-encrypted pool must be at least the same size as the previous GELI-encrypted pool. Two options exist to migrate data from a GELI-encrypted pool to a new ZFS-encrypted pool. The first method is to use `rsync` to copy the data between the pools. The second method is to use ZFS send/receive commands.
+It is not possible to *convert* an existing FreeNAS/TrueNAS 11.3 or earlier GELI-encrypted pool to use native ZFS encryption.
+
+## Migration from GELI
+
+Data can be *migrated* from the GELI-encrypted pool to a new ZFS-encrypted pool.
+Be sure to unlock the GELI-encrypted pool before attempting any data migrations.
+The new ZFS-encrypted pool must be at least the same size as the previous GELI-encrypted pool.
+Two options exist to migrate data from a GELI-encrypted pool to a new ZFS-encrypted pool: file transfer or ZFS send/receive.
+
+ZFS data migration can be a complicated process.
+You can ask for assistance in the [TrueNAS Community Forums](https://www.ixsystems.com/community/) or, if you have a support contract with iXsystems, [contact iX Support](/hub/additional-topics/support/#support-in-truenas-enterprise) for assistance.
+
+### File Transfer
+
+The first method is to use `rsync` or other file transfer mechanisms (`scp`, `cp`, `sftp`, `ftp`, `rdiff-backup`) to copy the data between the pools.
+Transferring your ACLs can be done with `getfacl` and `setfacl` if the chosen transfer method does not preserve ACLs.
```

Comment:
Let's rework this to show an example of using the UI to set up an rsync task for the migration. That way we'll have a substantive UI example and CLI example for folks of different comfort levels.